internal/pkg/cli/deploy/deploy.go

```diff
@@ -557,21 +608,37 @@ func (d *jobDeployer) DeployWorkload(in *DeployWorkloadInput) (ActionRecommender
 	return nil, nil
 }
 
-func (d *svcDeployer) deploy(in *DeployWorkloadInput, stackConfigOutput svcStackConfigurationOutput) error {
+func (d *workloadDeployer) generateCloudFormationTemplate(conf stackSerializer) (
+	*GenerateCloudFormationTemplateOutput, error) {
+	tpl, err := conf.Template()
+	if err != nil {
+		return nil, fmt.Errorf("generate stack template: %w", err)
+	}
+	params, err := conf.SerializedParameters()
+	if err != nil {
+		return nil, fmt.Errorf("generate stack template configuration: %w", err)
+	}
+	return &GenerateCloudFormationTemplateOutput{
+		Template:   tpl,
+		Parameters: params,
+	}, nil
+}
+
+func (d *svcDeployer) deploy(forceNewUpdate bool, stackConfigOutput svcStackConfigurationOutput) error {
```

Comment:
super nit: boolean parameters are usually considered a code smell if it's part of the _public_ API. Since it's not here, I don't care much 
https://martinfowler.com/bliki/FlagArgument.html

but if it's easy to refactor consider having two methods:
```go
func (d *svcDeployer) forceDeploy(out svcStackConfigOutput) error
func (d *svcDeployer) deploy(out svcStackConfigOutput) error
```
If it's a pain, leave it as is 👍

Reply:
That's a good point! But since it's not a public one and splitting it into two would add extra complexity. I'll just leave it as is.